admin/src/main/resources/io/buoyant/admin/js/logger-overrides.js

```diff
@@ -0,0 +1,26 @@
+$(function() {
+  $("nav#sidebar").remove();
+  $("#toggle").remove();
+
+  $("td>div.btn-group").addClass("pull-right");
+  $("thead>tr>th:eq(1)").addClass("pull-right");
+
+  var navbar = '<nav class="navbar navbar-inverse">' +
+  '      <div class="navbar-container">' +
+  '        <div class="navbar-header">' +
+  '          <a class="navbar-brand-img" href="/">' +
+  '            <img alt="Logo" src="/files/images/linkerd-horizontal-white-transbg-vectorized.svg">' +
+  '          </a>' +
+  '        </div>' +
+  '        <div id="navbar" class="collapse navbar-collapse">' +
+  '          <ul class="nav navbar-nav">' +
+  '            <li><a href="/delegator">dtab</a></li>' +
+  '            <li><a href="/admin/logging">logging</a></li>' +
+  '            <li><a href="https://linkerd.io/help/">help</a></li>' +
+  '          </ul>' +
+  '        </div>' +
+  '      </div>' +
+  '    </nav>"';
```

Comment:
Can we send this html down as a string inside a script tag? Like:
```
<script id="linkerd-navbar" type="application/json">{"navbarToInject": "<nav class=..."}</script>
```

This way ideally we can reuse the nav bar from other pages without duplication.